content/api/_index.md

```diff
@@ -21,6 +21,7 @@ and datasets.
 |:-----------------------------------------------------------------------------| :---: | :---: | :---: | :---: | :--- |
 | [GTFS Sverige 2](/api/trafiklab-apis/gtfs-sverige-2/)                        | GTFS | ✅ | ❌ | Entire Sweden | Dataset with all public transport, static and historical |
 | [GTFS Regional](/api/trafiklab-apis/gtfs-regional/)                          | GTFS(-RT) | ✅ | Some, [see details](/api/trafiklab-apis/gtfs-regional/#which-operators-are-covered-by-this-dataset) | [See details](/api/trafiklab-apis/gtfs-regional/#which-operators-are-covered-by-this-dataset) | Public Transport Datasets |
+| [SIRI](/api/trafiklab-apis/siri/)                                            | SIRI | ✅ | ❌ | Entire Sweden | Service Interface for Real Time Information |
```

Comment:
SIRI does not cover entire Sweden. Seeing that NeTEx is missing in this table, this row could be altered to be "NeTEx/SIRI", similar to how GTFS regional is one row for both static and realtime data